addons/l10n_nl/migrations/3.3/post-migrate_update_taxes.py

```diff
@@ -0,0 +1,58 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import api, SUPERUSER_ID
+
+
+def migrate(cr, version):
+    env = api.Environment(cr, SUPERUSER_ID, {})
+    goods_taxes = env['account.tax'].search([('name', '=', 'Verkopen export binnen EU (producten)')], limit=1)
+    services_taxes = env['account.tax'].search([('name', '=', 'Verkopen export binnen EU (diensten)')], limit=1)
+
+    old_plus_tax_tag_id = env['account.account.tag'].search(env['account.account.tag']._get_tax_tags_domain('3bl (omzet)', 'nl', '+'), limit=1).id
+    old_minus_tax_tag_id = env['account.account.tag'].search(env['account.account.tag']._get_tax_tags_domain('3bl (omzet)', 'nl', '-'), limit=1).id
+
+    goods_plus_tax_tag_id = env['account.account.tag'].search(env['account.account.tag']._get_tax_tags_domain('3bg (omzet)', 'nl', '+'), limit=1).id
+    goods_minus_tax_tag_id = env['account.account.tag'].search(env['account.account.tag']._get_tax_tags_domain('3bg (omzet)', 'nl', '-'), limit=1).id
+    services_plus_tax_tag_id = env['account.account.tag'].search(env['account.account.tag']._get_tax_tags_domain('3bs (omzet)', 'nl', '+'), limit=1).id
+    services_minus_tax_tag_id = env['account.account.tag'].search(env['account.account.tag']._get_tax_tags_domain('3bs (omzet)', 'nl', '-'), limit=1).id
+    insert_query_params = [
+        (goods_plus_tax_tag_id, goods_taxes.ids, old_plus_tax_tag_id, goods_taxes.invoice_repartition_line_ids.ids),
+        (services_plus_tax_tag_id, services_taxes.ids, old_plus_tax_tag_id, services_taxes.invoice_repartition_line_ids.ids),
+        (goods_minus_tax_tag_id, goods_taxes.ids, old_minus_tax_tag_id, goods_taxes.refund_repartition_line_ids.ids),
+        (services_minus_tax_tag_id, services_taxes.ids, old_minus_tax_tag_id, services_taxes.refund_repartition_line_ids.ids),
+    ]
+    insert_query_parts = []
+
+    for new_tax_tag_id, tax_ids, old_tax_tag_id, repartition_line_ids in insert_query_params:
+        insert_query_parts.append(
+            f"""
+                SELECT tag_aml_rel.account_move_line_id, {new_tax_tag_id}
+                FROM account_account_tag_account_move_line_rel tag_aml_rel
+                INNER JOIN account_move_line_account_tax_rel aml_at_rel ON aml_at_rel.account_move_line_id = tag_aml_rel.account_move_line_id
```

Comment:
```suggestion
                JOIN account_move_line_account_tax_rel aml_at_rel ON aml_at_rel.account_move_line_id = tag_aml_rel.account_move_line_id
```
It's the default, no need to specify it :nerd_face:

Reply:
Okey thanks :)